dashboard/app/models/peer_review.rb

```diff
@@ -280,17 +287,6 @@ def review_completed?
     (status == 'accepted' || status == 'rejected') || data?
   end
 
-  def create_escalated_duplicate
```

Comment:
This is only called once in our codebase, so I moved it into where it is called and (hopefully) clarified the overlap/differences between the two PeerReview objects (in the usual case -- only one PeerReview object in the new instructor review only case) that are created here.